src/dotnet/scripts/vendor/dotnet-install.sh

```diff
@@ -1295,6 +1379,11 @@ get_download_link_from_aka_ms() {
             return 1
         fi
 
+        sanitized_redirect_url=$(sanitize_redirect_url "$aka_ms_download_link")
```

Comment:
Do we know when and how the aka.ms redirects for the dotnet downloads will be updated? @bamurtaugh Do you have a contact to check with?

@sliekens Maybe the script could check if the redirect still points to `azureedge.net` and only in that case run the sanitize step? That might be more robust towards changes of the redirects.

Reply:
https://github.com/dotnet/core/issues/9671 says about the aka.ms redirects: `We expect these links to be changed in early January.` So this should keep working without modification.

Reply:
Seeing that you copied https://github.com/dotnet/install-scripts/blob/main/src/dotnet-install.sh, I guess that might see another update?

Reply:
Yes.

- We will update the file again in the first half of January. Please grab copies.
- We will be providing users with guidance on how to do freshness checks on when to grab new copies.

Related: https://github.com/dotnet/install-scripts/issues/566